meson_options.txt

```diff
@@ -91,6 +91,10 @@ option('libarchive',
   type: 'feature',
   description: 'libarchive support',
 )
+option('libdrm',
+  type: 'feature',
+  description: 'libdrm support',
+)
```

Comment:
IMO we should drop `plugin_amdgpu` meson option if adding this and make _it_ automagic.

Reply:
That's basically what we have, no? or do you mean we should do `s/plugin_amdgpu/libdrm_amdgpu`?

Reply:
I think if we add a toplevel `libdrm` meson tri-state feature then we don't need a toplevel `plugin_amdgpu` meson tristate feature anymore.  Just let whether `libdrm` is enabled control whether we get `plugin_amdgpu`

Reply:
Hmm, I'm not sure about this; if you compile libdrm with `-Damdgpu=disabled` then `libdrm.pc` exists, but not `libdrm_amdgpu.pc`

Reply:
Ahhhh true.  OK.